app/src/main/java/com/example/geoquiz/MainActivity.kt

```diff
@@ -39,10 +43,20 @@ class MainActivity : AppCompatActivity() {
     private fun setOnClickListeners() {
         trueButton.setOnClickListener {
             checkAnswer(true)
+            questionBank[currentIndex].answeredQuestion = true
+            answeredQuestionCheck()
+            if (questionBank[questionBank.size - 1].answeredQuestion == true) {
```

Comment:
`if` statement can be simplified with `questionBank[questionBank.size - 1].answeredQuestion`